### ISSUE TITLE
FFTW preset, fftwf_iodim and fftwf_iodim64 missing array allocators

### ISSUE THREAD

**Issue by hageldave** [bytedeco/javacpp-presets#523]
Hello, I previously reported missing definitions of fftw_iodim struct in #466 which was already fixed.
Now I just noticed that the float version of this class does not  provide the same interface.

Here is the definition of fftw_iodim:
```java
@Name("fftw_iodim_do_not_use_me") public static class fftw_iodim extends Pointer {
    static { Loader.load(); }
    /** Default native constructor. */
    public fftw_iodim() { super((Pointer)null); allocate(); }
    /** Native array allocator. Access with {@link Pointer#position(long)}. */
    public fftw_iodim(long size) { super((Pointer)null); allocateArray(size); }
    /** Pointer cast constructor. Invokes {@link Pointer#Pointer(Pointer)}. */
    public fftw_iodim(Pointer p) { super(p); }
    private native void allocate();
    private native void allocateArray(long size);
    @Override public fftw_iodim position(long position) {
        return (fftw_iodim)super.position(position);
    }

     public native int n(); public native fftw_iodim n(int n);                     /* dimension size */
     public native int is(); public native fftw_iodim is(int is);			/* input stride */
     public native int os(); public native fftw_iodim os(int os);			/* output stride */
}
```

and here the definition of fftwf_iodim:
```java
@Opaque public static class fftwf_iodim extends Pointer {
    /** Empty constructor. Calls {@code super((Pointer)null)}. */
    public fftwf_iodim() { super((Pointer)null); }
    /** Pointer cast constructor. Invokes {@link Pointer#Pointer(Pointer)}. */
    public fftwf_iodim(Pointer p) { super(p); }
}
```

I think there's something missing but I could be wrong and maybe there is another way of using the fftwf_iodim class to allocate an array of it and set n, is, os.

**Comment by saudet** [comment]
Yes, they are missing, but they are just typedefs of the same struct. I'll make the necessary changes later today. Thanks for reporting!

**Comment by saudet** [comment]
For now, we can do something like this:
```java
fftw_iodim dims = new fftw_iodim(n);
// ...
fftwf_iodim fdims = new fftwf_iodim(dims);
```

**Comment by hageldave** [comment]
nice, pretty fast support on here :)

**Comment by saudet** [comment]
FYI, released with version 1.4.1. Enjoy!
